### PR TITLE
Incorporated blobstore into my my form submission. Content uploads th…

### PR DESCRIPTION
**Pull request by Jaxobc**
…rough my blobstore URL

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -69,40 +69,43 @@ <h2>Ballers</h2>
...
           <p>Required Format: Text without whitespaces at the beginning or end.</p>
           <label for="name">Name:</label><br>
           <input type="text" required pattern="\S(.*\S)?" maxlength="20" name="name"  textarea placeholder="Enter comment here..."><br>
           <label for="comment">Comment:</label><br>
           <input type="text" required pattern="\S(.*\S)?" maxlength="120" name="text-input" textarea placeholder="Enter comment here..."><br><br>
+          <label for="image">Image:</label><br>
+          <input type="file" name="image">
+          <br/><br/>
           <label for="commentLimit">Choose the amount of comments you want to see:</label>
-          <input type="submit" value="Submit"><br><br>
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/index.html:99]: Try css paddings instead of relying on "<br".

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -73,3 +75,15 @@ function drawChart(limitMap) {
   const chart = new google.visualization.PieChart(document.getElementById('chart-div'));
   chart.draw(data, options);
 }
+
+function fetchBlobUrlAndShowForm() {
+  fetch('/blobstore-url')
+      .then((response) => {
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/script.js:81]: You can do one line lambda such as `then(response => response.text())`

Comment (Jaxobc) [reply portfolio/src/main/webapp/script.js:81]: Alright sure Tong.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (Jaxobc) [reply portfolio/src/main/webapp/index.html:99]: Okay sure, I'll remove all <br> tags that I'm currently using for stylistic purposes.

**Pull request closed without merging**